okhttp-ws-tests/src/test/java/com/squareup/okhttp/internal/ws/RealWebSocketTest.java

```diff
@@ -247,13 +247,6 @@
         clientListener.assertClose(1000, "Hello!");
 
         try {
-          sink.writeUtf8("lo!").emit(); // No writing to the underlying sink.
```

Comment:
Message writes are single-threaded by policy and thus don't take locks to check whether the larger writer is closed. If this write was larger than a segment it would fail, however.